tests/calibration/calibration_test.py

```diff
@@ -75,7 +80,16 @@ def test_reg_l2_called(self):
                                   double_fit=False, workers=0)
         self.calibrator.l2_reg.assert_called()
 
+    def test_weight_assignment(self):
```

Comment:
This test could be merge with the previous one if we omit `init_model` from the class. It was intended to do same thing